src/commands/flutter-symbols/__tests__/upload.test.ts

```diff
@@ -42,10 +42,6 @@ const cliVersion = require('../../../../package.json').version
 const fixtureDir = './src/commands/flutter-symbols/__tests__/fixtures'
 
 describe('flutter-symbol upload', () => {
-  beforeAll(() => {
-    jest.clearAllMocks()
-  })
```

Comment:
Clearing the mocks is already set in `jest.config.js`: https://github.com/DataDog/datadog-ci/blob/339759d12420442a32d55deada040b1afdd79860/jest.config.js#L6-L7